admin_guide/managing_networking.adoc

```diff
@@ -85,6 +85,42 @@ can now access any pods and services in the cluster and vice versa.
 Alternatively, instead of specifying specific project names, you can use the
 `--selector=<project_selector>` option.
 
+[[admin-guide-disabling-hostname-collision]]
+== Disabling Host Name Collision Prevention For Ingress Objects
+
+In {product-title}, host name collision prevention for routes and ingress
+objects is enabled by default. This means that the host name in a route or
+ingress object can only be set on creation and not edited afterwards. Disabling
+host name collision prevention lets you edit a host name after creation.
```

Comment:
It might be important to mention that this only enables editing of hostnames for ingress objects. This controller has no effect on route objects